src/Homepage.js

```diff
@@ -0,0 +1,87 @@
+import { useState, useEffect } from "react";
+
+const Homepage = () => {
+    const [amountOfSnippets, setamountOfSnippets] = useState('')
+    const [text, setText] = useState('')
+    const [faild, setFaild] = useState('')
+    const url = "http://localhost:7000/api/generateText";
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
```

Comment:
No need to preventdefault since this button isn't inside a form element, and this button isn't type of submit.